src/input/keyboard_bindings.rs

```diff
@@ -0,0 +1,11 @@
+use std::collections::HashMap;
+use glutin::event::VirtualKeyCode;
+
+pub enum Actions {
+    MoveForward,
```

Comment:
this is nice tho, but I think the moves are not actually actions, something something axis again